### ISSUE TITLE
void-variable read-symbol-positions-list error on emacs-29 HEAD

### ISSUE THREAD

**Issue by jguenther** [Wilfred/elisp-refs#35]
looks like `read-with-symbol-position` and `read-symbol-positions-list` were removed in emacs commit https://github.com/emacs-mirror/emacs/commit/dfae76c9915454f9ef0885e1bac160a2c480e1d1

```
if: Unexpected error whilst reading ~/.emacs.d/elpa/company-20220110.2248/company.el position 2497: (void-variable read-symbol-positions-list)
```

I took a quick look and I'm not sure what the replacement would be for this, but I'd assume there is one (`read-positioning-symbols` maybe?)

thx for this great package btw _!_

**Comment by OlavStornes** [comment]
Happens to me as well, attempted on two versions: `29.0.50.153980` and  `29.0.50.153998`

Error is occuring when using `helpful`-functions, which, as i understand, has elisp-refs as a dependency.
Stacktrace:
```
Debugger entered--Lisp error: (error "Unexpected error whilst reading /usr/share/emacs/2...")
  error("Unexpected error whilst reading %s position %s: %s" "/usr/share/emacs/29.0.50/lisp/delsel.el" 2653 (void-variable read-positioning-symbols))
  elisp-refs--read-all-buffer-forms(#<buffer  *refs-/usr/share/emacs/29.0.50/lisp/delsel.el*-587138>)
  elisp-refs--read-and-find(#<buffer  *refs-/usr/share/emacs/29.0.50/lisp/delsel.el*-587138> delete-selection-mode elisp-refs--variable-p)
  #f(compiled-function (buf) #<bytecode 0x659208fd8bf9237>)(#<buffer  *refs-/usr/share/emacs/29.0.50/lisp/delsel.el*-587138>)
  elisp-refs--search-1((#<buffer  *refs-/usr/share/emacs/29.0.50/lisp/delsel.el*-587138>) #f(compiled-function (buf) #<bytecode 0x659208fd8bf9237>))
  helpful--reference-positions(delete-selection-mode nil #<buffer  *refs-/usr/share/emacs/29.0.50/lisp/delsel.el*-587138>)
  helpful--calculate-references(delete-selection-mode nil "/usr/share/emacs/29.0.50/lisp/delsel.el")
  helpful-update()
  helpful--update-and-switch-buffer(delete-selection-mode nil)
  helpful-variable(delete-selection-mode)
  funcall-interactively(helpful-variable delete-selection-mode)
  command-execute(helpful-variable)
```

**Comment by jacksondm33** [comment]
I'm pretty sure this commit is the cause of the issue:
https://git.savannah.gnu.org/cgit/emacs.git/commit/?id=dfae76c9915454f9ef0885e1bac160a2c480e1d1

**Comment by Wilfred** [comment]
Fixed in #36.
